### ISSUE TITLE
[8.0.4] parse throws while using postcss-safe-parser

### ISSUE THREAD

**Issue by AviVahl** [postcss/postcss#1411]
Appreciate the fix for #1407. Those failing tests are now passing on our end. :)

There's a new regression with 8.0.4 when (postcss-safe-)parsing the following:
```
:import {
    -st-from: "b/index.st.css"
    -st-default: B;
}
.root {
    -st-extends: B;
}
```
I get the following Error thrown:
```
Uncaught TypeError: Cannot read property 'line' of null
    at SafeParser.getPosition (/home/avi/projects/stylable/node_modules/postcss/lib/parser.js:382:17)
    at SafeParser.init (/home/avi/projects/stylable/node_modules/postcss/lib/parser.js:390:19)
    at SafeParser.decl (/home/avi/projects/stylable/node_modules/postcss/lib/parser.js:166:10)
```

**Comment by ai** [comment]
Working on it

**Comment by AviVahl** [comment]
Thanks. I believe the missing semi-colon is the curl-pit. Adding it fixed the exception.

**Comment by ai** [comment]
Fixed in `postcss-safe-parser` 5.0.1
